commands/commands_test.go

```diff
@@ -52,7 +53,11 @@ func getTestStore() (libmachine.Store, error) {
 
 	os.Setenv("MACHINE_STORAGE_PATH", tmpDir)
 
-	return libmachine.NewFilestore(tmpDir, hostTestCaCert, hostTestPrivateKey), nil
+	return persist.Filestore{
```

Comment:
the `persist.Filestore` thing turns out a LOT cleaner than `store.Filestore` which is part of why I want to rename `host` module.